crates/sui-indexer/src/lib.rs

```diff
@@ -203,7 +203,8 @@ pub async fn multi_get_full_transactions(
             SuiTransactionResponseOptions::new()
                 .with_input()
```

Comment:
It's quite wasteful to request both `input` and `raw_input` from fullnode, since you can derive the latter from the former, can you add a MUSTFIXD(gegao) comment to avoid fetching the latter somewhere?

Reply:
derive input from raw_input, right? will add a MUSTFIX